### PR TITLE
remove codesign script for cocoapod only installation

### PR DESCRIPTION
**Pull request by MikeSilvis**

### CODE REVIEW

File: SquareInAppPaymentsSDK.podspec [hunk 1]
```diff
@@ -30,4 +30,9 @@ Pod::Spec.new do |s|
   s.source_files = 'SquareInAppPaymentsSDK.framework/**/*.{h,hh}'
   s.public_header_files = 'SquareInAppPaymentsSDK.framework/Headers/*.{h,hh}'
   s.vendored_frameworks = 'SquareInAppPaymentsSDK.framework'
+  s.script_phase = {
+      name: 'Remove codesign script',
```
Comment (kgleong) [inline SquareInAppPaymentsSDK.podspec]: Nit: can we rename this to something like `Remove Square In-App Payments SDK setup script for Cocoapod installs`

**Comment by kgleong** [comment]
remove first commit?

File: SquareInAppPaymentsSDK.podspec [hunk 2]
```diff
@@ -30,4 +30,9 @@ Pod::Spec.new do |s|
   s.source_files = 'SquareInAppPaymentsSDK.framework/**/*.{h,hh}'
   s.public_header_files = 'SquareInAppPaymentsSDK.framework/Headers/*.{h,hh}'
   s.vendored_frameworks = 'SquareInAppPaymentsSDK.framework'
+  s.script_phase = {
+      name: 'Remove Square In-App Payments SDK setup script for Cocoapod installs',
+      script: 'if ! (grep -i SquareInAppPaymentsSDK.framework/setup ../*/project.pbxproj) ; then rm -f SquareInAppPaymentsSDK/SquareInAppPaymentsSDK.framework/setup ; fi',
```
Comment (kgleong) [inline SquareInAppPaymentsSDK.podspec]: Can we add a comment here like:

```
# Do not remove the setup script if the host app contains a build phase that executes it.
```

File: SquareInAppPaymentsSDK.podspec [hunk 3]
```diff
@@ -30,4 +30,10 @@ Pod::Spec.new do |s|
   s.source_files = 'SquareInAppPaymentsSDK.framework/**/*.{h,hh}'
   s.public_header_files = 'SquareInAppPaymentsSDK.framework/Headers/*.{h,hh}'
   s.vendored_frameworks = 'SquareInAppPaymentsSDK.framework'
+  s.script_phase = {
+      name: 'Remove Square In-App Payments SDK setup script for Cocoapod installs',
+      # Only remove the setup script if the host app does not contain a build phase that executes it.
```
Comment (kgleong) [inline SquareInAppPaymentsSDK.podspec]: # Only remove the setup script if the host app does not contain a build phase that executes it `or the project file cannot be found.`

**Comment by MikeSilvis** [comment]
Closing in favor of updates coming to the SDK

**Pull request closed without merging**